### PR TITLE
[PR] Introduce UGRD report download shortcode

### PR DESCRIPTION
**Pull request by jeremyfelt**

### CODE REVIEW

**Comment by jeremyfelt** [comment]
@philcable #reviewmerge?

File: js/ugrd-dropdown.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+( function( $) {
```
Comment (philcable) [inline js/ugrd-dropdown.js]: Missing space after `$`

**Review by philcable — APPROVED** [review]
I left one inline comment about a missing space, otherwise looks great!

File: js/ugrd-dropdown.js (hunk 1 above)
Comment (jeremyfelt) [reply js/ugrd-dropdown.js]: haha, that punk. I still have JS linting off because of the other script. :) Thanks!

**Pull request merged**